index.bs

```diff
@@ -0,0 +1,139 @@
+<pre class='metadata'>
+Title: Audio Session
+Shortname: audio-session
+Level: None
+Status: w3c/ED
+Group: mediawg
+TR: https://www.w3.org/TR/audio-session/
+Repository: w3c/audio-session
+URL: https://w3c.github.io/audio-session/
+Editor: Youenn Fablet, Apple https://www.apple.com/, youenn@apple.com, w3cid 96458
+Editor: Alastor Wu, Mozilla https://www.mozilla.org, alwu@mozilla.com, w3cid 92198
+Abstract: This API defines an API surface for manipulating the bits on
```

Comment:
manipulating the bits on... ?

Reply:
This part seems not finished yet.

Reply:
Right, updated to 
`This API defines an API surface for controlling how audio is rendered and interacts with other audio playing applications`